hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestAuditLogger.java

```diff
@@ -231,6 +232,30 @@ public void testAuditLoggerWithSetPermission() throws IOException {
     }
   }
 
+  @Test
+  public void testAuditLoggerWithCreateFlag() throws IOException {
+    final Configuration conf = new HdfsConfiguration();
+    final MiniDFSCluster cluster = new MiniDFSCluster.Builder(conf).build();
+    LogCapturer auditlog = LogCapturer.captureLogs(FSNamesystem.auditLog);
+    try {
+      cluster.waitClusterUp();
+      auditlog.clearOutput();
+      final FileSystem fs = cluster.getFileSystem();
+      final Path p = new Path("/debug.log");
+      FSDataOutputStream stm = fs.create(p);
+      assertTrue(auditlog.getOutput().contains("create(options"));
```

Comment:
can we test a few more cases? e.g., with or without flag, and check whether the content of the flag is actually logged correctly.

Reply:
thanks for your suggestion.
I will modify it.